package.json

```diff
@@ -153,16 +153,17 @@
     "restoreMocks": true
   },
   "dependencies": {
-    "@reach/auto-id": "^0.18.0",
     "@restart/ui": "^1.0.1",
     "@tailwindcss/forms": "^0.5.0",
+    "@types/react": "^18.2.55",
```

Comment:
out of curiosity, why didn't we need this package before?

Reply:
removing